PYME/experimental/triangle_mesh_utils.c

```diff
@@ -72,6 +72,8 @@ static void update_single_vertex_neighbours(int v_idx, halfedge_t *halfedges, vo
     vertex_t *vertices = (vertex_t*) vertices_;
     face_t *faces = (face_t*) faces_;
 
+    if (v_idx == -1) return;
```

Comment:
should we silently ignore here, or flag an error?

Reply:
Flag an error.